ow_plexil/plugin.xml

```diff
@@ -0,0 +1,14 @@
+<library path="src">
+  <class name="PLEXIL Planner" type="rqt_plexil_planner.rqt_plexil_planner.PlexilPlanner" base_class_type="rqt_gui_py::Plugin">
+    <description>
+      A simple GUI that selects PLEXIL plans to be run on the ow_simulator.
```

Comment:
"to be run by the ow_plexil node" is more accurate, since I'm building an interface to also run plans on the owlat_sim simulator.